crates/tuono/src/scaffold_project.rs

```diff
@@ -93,15 +107,13 @@ pub fn create_new_project(folder_name: Option<String>, template: Option<String>)
 
     if new_project_files.is_empty() {
         eprintln!("Error: Template '{template}' not found");
-        println!("Hint: you can view the available templates at https://github.com/tuono-labs/tuono/tree/main/examples");
-        std::process::exit(1);
+        exit_with_error(&format!("Error: Template '{template}' not found. Hint: you can view the available templates at https://github.com/tuono-labs/tuono/tree/main/examples"));
```

Comment:
Better to have two separate calls IMHO:

```suggestion
        eprintln!("Error: Template '{template}' not found");
        exit_with_error(&format!("Hint: you can view the available templates at https://github.com/tuono-labs/tuono/tree/main/examples"));
```

Better development experience having the two sentences on two different rows